knife/lib/chef/knife/bootstrap.rb

```diff
@@ -32,7 +32,7 @@ class Bootstrap < Knife
       include DataBagSecretOptions
       include LicenseAcceptance::CLIFlags::MixlibCLI
 
-      SUPPORTED_CONNECTION_PROTOCOLS ||= %w{ssh winrm}.freeze
+      TRANSPORTS_BLACKLIST ||= %w{train-core train-aws train-azure train-habitat train-rest}.freeze
```

Comment:
shouldn't we have an allow list for `%w{ssh winrm}` that we `|` with `transports_available` for the result of `supported_connection_protocols`?

Reply:
Line 1206 - that exact thing is there.